buildconfig/Jenkins/Conda/download-and-install-mambaforge

```diff
@@ -10,12 +10,17 @@
 EXPECTED_MAMBAFORGE_PATH=$1
 EXPECTED_CONDA_PATH=$2
 CLEAN_BUILD=$3
+MAMBAFORGE_VERSION=4.11.0-0
+
+LINUX_SHA256="49268ee30d4418be4de852dda3aa4387f8c95b55a76f43fb1af68dcbf8b205c3  Mambaforge-4.11.0-0-Linux-x86_64.sh"
```

Comment:
I'm guessing you generated these, so just want to check, doesn't `sha256sum` stick an `*` before the filename like:
`*Mambaforge-4.11.0-0-Linux-x86_64.sh`

Reply:
I think I actually copied them from the download location, for example:  `https://github.com/conda-forge/miniforge/releases/download/4.11.0-4/Mambaforge-4.11.0-4-MacOSX-x86_64.sh.sha256` This file has a ./ in front of it because that is what they passed to the sha256 command, whilst I pass in this script just the file name and thus get a slightly different output.